homeassistant/helpers/entity_registry.py

```diff
@@ -394,8 +394,8 @@ def async_is_registered(self, entity_id: str) -> bool:
 
     @callback
     def async_get(self, entity_id: str) -> RegistryEntry | None:
```

Comment:
This param name should change since it can be two things now.

Reply:
Good point, fixed